salesforce/latest/using/community.md

```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
 
       ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
 
-  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
\ No newline at end of file
+  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
+  
+## Community Visual Force Page Configuration and Setup
+
+To expose the connector in a Salesforce Community (Digital Workspace)we need to add a visual force page for the connector. 
+Visual Force Page must be added to your Salesforce organization using the following pattern:
```

Comment:
There are no spaces in Visualforce

Reply:
and to keep concepts consistent it should be called a template and not a pattern

Reply:
updated